### DISCUSSION TITLE
weekly/issue-246

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#5] [General]
# weekly/issue-246

阮一峰的技术周刊

https://ruanyf-weekly.plantree.me/weekly/issue-246

**Comment by terminalqo** [comment]
#### 早就该有这了东西了。 老阮以后周五发布就丢个link 就完事了好吗。 
#### 微信公众号那阅读体验。。什么？微信公众号有阅读体验？

**Comment by dijianze** [comment]
# 阅读体验很好

**Comment by meixiaoliang** [comment]
好

**Comment by BrotaOne** [comment]
每期的评论都是共享的？

**Comment by sunbingfeng** [comment]
这个看着舒服多了，赞。不过自动构建有延迟啊，还看不了最新一期

**Comment by lulala1024** [comment]
终于等到了。感恩！

**Comment by MuLoo** [comment]
比原站漂亮👍。
提个小建议哈，左右两边的导航栏可以小一点，或者允许隐藏，注意力应该集中在内容区域

**Comment by iamflowerdog** [comment]
good,开源万岁

**Comment by kathleencarn** [comment]
为什么我的不显示图片，是网速太慢没加载出来吗

**Comment by exposir** [comment]
连图片都不支持现实 不知道你们夸啥呢  再说流量都跑这来了是不是对阮老师也有影响？

**Reply by debuginn** [reply]
阮老师计划迁移静态网站了，感觉在做了 后续支持的话就比较好呢

**Reply by debuginn** [reply]
主要是阮老师的个人站点顶不住黑客的恶意进攻 哈哈哈

**Reply by exposir** [reply]
现在体验很好 很赞

**Comment by Qusunyee** [comment]
nice

**Comment by debugly** [comment]
Good，页面加载很快。

**Comment by hjg66-5** [comment]
咱们这个评论可以想极客时间那样对具体一行进行评论么

**Comment by tr4c3** [comment]
赞，这个实现太好了。

**Comment by maincooss** [comment]
爱死了,阅读体验超棒的.

**Comment by Chivier** [comment]
请问有 RSS 吗？

**Reply by plantree** [reply]
可以参考阮老师网站的RSS：https://www.ruanyifeng.com/blog/atom.xml

**Comment by Alwayfeels** [comment]
支持一下，全文搜索很好用（可惜 Algolia 是闭源的）

**Reply by akarigooseo1** [reply]
Odoo is one of the best Marketing Inventory Software in the UK. This Odoo Inventory software helps small businesses to manage their inventory effectively.

**Reply by akarigooseo1** [reply]
https://akarigo.com/marketing-inventory-software-uk

**Reply by plantree** [reply]
Awesome! You can submit [here](https://github.com/ruanyf/weekly/issues).

**Comment by lzfgit-byte** [comment]
vercel 现在直连不行了，换netlify吧，这个能直连

**Reply by plantree** [reply]
正在尝试修复！

**Reply by lzfgit-byte** [reply]
vercel 要是能修好的话，出篇文章吧。

**Reply by plantree** [reply]
短线方案 先迁移到Netlify了......

**Comment by spboxer3** [comment]
已经三週（两篇）没有同步文章了，是不是要检查一下程序还有没有正常运作
